### PR TITLE
[BPF] split calico_tc_skb_accepted_entrypoint

### PR DESCRIPTION
**Pull request by tomastigera**
Because of insufficient stack space and hitting the limit way too often in debug programs, handling new flows (and creating conntrack entries) is split off into a separate program.

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
TBD
```

## Reminder for the reviewer

Make sure that this PR has the correct labels and milestone set.

Every PR needs one `docs-*` label.

- `docs-pr-required`: This change requires a change to the documentation that has not been completed yet.
- `docs-completed`: This change has all necessary documentation completed.
- `docs-not-required`: This change has no user-facing impact and requires no docs.

Every PR needs one `release-note-*` label.

- `release-note-required`: This PR has user-facing changes. Most PRs should have this label.
- `release-note-not-required`: This PR has no user-facing changes.

Other optional labels:

- `cherry-pick-candidate`: This PR should be cherry-picked to an earlier release. For bug fixes only.
- `needs-operator-pr`: This PR is related to install and requires a corresponding change to the operator.

### CODE REVIEW

File: felix/bpf/ut/bpf_prog_test.go
Comment (tomastigera) [inline felix/bpf/ut/bpf_prog_test.go:221]: this and other ipv6 values are gone in further ipv6 patches

**Review by fasaxc — APPROVED** [review]
Looks good but not sure I would be able to spot any small issues amongst the large code move!

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Pull request merged**
